configs/azure.yaml

```
@@ -1,10 +1,14 @@
 test: false
+env: azure
 run_name: ehr_preprocess
 paths:
-  output_dir: outputs
-  # output_dir: ../../data/formatted_data/medication_diagnosis
-  file_type: parquet
-  
+  output_dir: outputs/new_ehr_preprocess
+  file_type: parquet 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)